### DISCUSSION TITLE
[Springboot Template] Can t run the tests

### DISCUSSION THREAD

**Discussion by alexBonadei** [mia-platform/community#337] [Q&A] [answered]
### Which section?

Marketplace

### Which is the version?

latest

### Description

Hi! 
I created a service using the Springboot Template.

If I run the command `mvn test` as explained in the README file, the tests are not executed. Those tests are the ones that come with the template (CPRequestTest.java and StatusControllerTest.java)

![Screenshot from 2023-11-15 14-16-20](https://github.com/mia-platform/community/assets/67758444/8c6c4b4e-9ce3-4fc1-8f3c-07c148c11558)

Am I missing something?
Thanks in advance.

**Comment by fredmaggiowski** [comment, marked as answer]
Hi, I'd open an issue on the template itself rather than a discussion here

**Reply by alexBonadei** [reply]
Yes, you're right.

Actually I realized that there is already an issue regarding this bug. https://github.com/mia-platform-marketplace/SpringBoot-Custom-Plugin-Template/issues/119
